src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -164,11 +164,10 @@ public static void showPersonAtIndex(Model model, Index targetIndex) {
      */
     public static void showTaskAtIndex(Model model, Index targetIndex) {
         assertTrue(targetIndex.getZeroBased() < model.getFilteredTaskList().size());
-
         Task task = model.getFilteredTaskList().get(targetIndex.getZeroBased());
         final String taskName = task.getName();
-        model.updateFilteredTaskList(new TaskNameContainsKeywordsPredicate(Arrays.asList(taskName)));
```

Comment:
`taskName` should be split into separate words because each item in the list passed to `Predicate` only accept a single-word item.